code/modules/client/client_colour.dm

```diff
@@ -108,6 +108,10 @@
 	colour = "#cccccc"
 
 
+/datum/client_colour/eye_colour/monochrome
+	colour = COLOR_MONOCHROME
+	priority = 1
+
 /datum/client_colour/monochrome
-	colour = list(rgb(77,77,77), rgb(150,150,150), rgb(28,28,28), rgb(0,0,0))
+	colour = COLOR_MONOCHROME
```

Comment:
Define the values here if you want but you should not be defining the whole list